.changeset/rich-crews-repeat.md

```diff
@@ -0,0 +1,5 @@
+---
+"openapi-typescript": patch
```

Comment:
Whoops! Didn’t catch this was for the wrong package. Should be `openapi-fetch`.

Reply:
Ohh... crap. Sorry. I'll issue a fix PR.

Reply:
I see it has been already released. 😢

Reply:
Yours was released correctly! 🙂 https://github.com/openapi-ts/openapi-typescript/pull/1889

The `openapi-typescript` release was for an actual patch from another PR: https://github.com/openapi-ts/openapi-typescript/pull/1890